.env

```diff
@@ -0,0 +1,13 @@
+MYSQLDB_ROOT_PASSWORD=1234qwer
```

Comment:
it's better to create .env.sample file with these properties (but without values, just names of properties)
add .env to gitignore file
remove .env from git